examples/community/README.md

```diff
@@ -3406,6 +3420,7 @@ pipe(prompt=prompt, rp_args=rp_args)
 ### Optional Parameters
 
 - `save_mask`: In `Prompt` mode, choose whether to output the generated mask along with the image. The default is `False`.
+- `base_ratio`: Used with `ADDBASE`. Sets the ratio of the base prompt; if base ratio is set to 0.2, then resulting images will consist of `20%*BASE_PROMPT + 80%*REGION_PROMPT`
```

Comment:
Could maybe also specify the reference that introduced this technique?

Reply:
This technique was originally implemented in [hako-mikan's regional-prompter](https://github.com/hako-mikan/sd-webui-regional-prompter), but not in the current community pipeline.
So, I don't think it's necessary to specify the reference again.